go/core/servers.go

```diff
@@ -122,7 +122,13 @@ func (s *devServer) handleRunAction(w http.ResponseWriter, r *http.Request) erro
 		// Stream results are newline-separated JSON.
 		callback = func(ctx context.Context, msg json.RawMessage) error {
 			_, err := fmt.Fprintf(w, "%s\n", msg)
-			return err
+			if err != nil {
+				return err
+			}
+			if f, ok := w.(http.Flusher); ok {
+				f.Flush()
```

Comment:
I'm surprised `Transfer-Encoding: chunked` header is not required... but appears to be the case: https://stackoverflow.com/questions/26769626/send-a-chunked-http-response-from-a-go-server